src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
And is more likely to be incorrect than the individual file due to all the activity there.  If we cannot trust these individual cert file contents then why are we trusting the CA bundle of certs content?  Trusting on and not the other is pointless.